distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -116,6 +116,10 @@ Data can be ingested via ${catalog-ui}.
 . Drag and drop file(s) or click to open a navigation window.
 . After selecting the file(s) to be uploaded, select *Start* to begin uploading.
 
+Files will be processed individually with a visual cues and messaging indicating the successful resolution of each ingest operation.
+If an ingest operation is unsuccessful, the user will be notified with a message on that specific product.
```

Comment:
This section avoids the use of the word "ingest", probably because it is UI documentation and the UI uses the word "upload" (I think???). Also, in the interest of parallelism, "product" should probably be "file(s)". Here is my suggestion:

"If ${branding} cannot process a file(s), ${branding}" notifies the user that the upload for that file failed"

Reply:
It looks like ingest is used more than upload in this section so maybe we should swap the `upload` for `ingest`.

Reply:
Then again... we have a ui button named `Upload` 🤔